src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AzureFunction/InProcessFunctionDetails.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace NewRelic.Providers.Wrapper.AzureFunction;
+
+public class InProcessFunctionDetails
+{
+    public string Trigger { get; set; }
+    public string TriggerTypeName { get; set; }
+    public bool IsWebTrigger => Trigger == "http";
+    public string FunctionName { get; set; }
+    public ServiceBusTriggerDetails ServiceBusTriggerDetails { get; set; }
+}
+
+public  class ServiceBusTriggerDetails
+{
+    public string QueueName { get; set; }
+    public string TopicName { get; set; }
+    public string SubscriptionName { get; set; }
+    public string Connection { get; set; } // not a connection string; just the name of the appSetting that contains the connection string
+
+    public ServiceBusDestinationType DestinationType => !string.IsNullOrEmpty(QueueName) ? ServiceBusDestinationType.Queue : ServiceBusDestinationType.Topic;
```

Comment:
I am not seeing any references to this, can you point me in the right direction?

Reply:
Actually, I don't think anything in the `ServiceBusTriggerDetails` class is used at all so I'll get rid of it. Thanks for pointing it out!